carina-api/src/main/java/com/qaprosoft/carina/core/foundation/api/AbstractApiMethod.java

```diff
@@ -100,7 +100,8 @@ private void init(Class clazz) {
         }
         if (typePath.contains(":")) {
             methodType = HttpMethodType.valueOf(typePath.split(":")[0]);
-            methodPath = typePath.split(":")[1];
+            methodPath = typePath.split(methodType + ":")[1];
```

Comment:
@yrozhenko let's make it some more obvious here. for example instead of split you can use StringUtils.substringAfter(). This way logic will be more clear for understanding